src/api/Authentication/LegacyApiAuthenticationMiddleware.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Security.Claims;
+
+namespace BDMS.Authentication;
+
+public class LegacyApiAuthenticationMiddleware(ILogger<LegacyApiAuthenticationMiddleware> logger) : IMiddleware
+{
+    private readonly ILogger<LegacyApiAuthenticationMiddleware> logger = logger;
+
+    public async Task InvokeAsync(HttpContext context, RequestDelegate next)
+    {
+        Claim? userName;
```

Comment:
REQ: Die Variable in die `if`-Anweisung verschieben, da sie ausserhalb nicht benötigt wird.